riemann_client/client.py

```diff
@@ -152,6 +153,9 @@ def create_dict(event):
                 value = list(value)
             elif descriptor.name == 'attributes':
                 value = dict(((a.key, a.value) for a in value))
+            elif descriptor.type == \
+                    google.protobuf.descriptor.FieldDescriptor.TYPE_FLOAT:
+                value = round(value, 1)
```

Comment:
Using the C++ backend for protobuf results in floating point precision loss. Applications should be able to tolerate the IEEE 754 FP spec essentially.